torchaudio/datasets/commonvoice.py

```diff
@@ -0,0 +1,92 @@
+import os
+
+import torch.utils.data as data
+
+import torchaudio
+from torchaudio.datasets.utils import (
+    download,
+    extract,
+    shuffle,
+    unicode_csv_reader,
+    walk,
+)
+
+
+def load_commonvoice_item(line, header, path, folder_audio):
```

Comment:
This function is only called once. Is there an anticipation for it to be used more widely in the future? Otherwise I suggest you inline it given that it's quite short.

Reply:
In all of those, I used a `load_thisdataset_item`, so that I decouple the "load this particular item" from `__getitem__` or `__iter__`` etc, and make the class easy to convert from an list to an iterable or something else. Thoughts?